os-eap-sso/added/keycloak.sh

```diff
@@ -85,22 +75,22 @@ function configure_keycloak() {
     if [ -n "$SSO_SERVICE_URL" ]; then
       sso_service="$SSO_SERVICE_URL"
     fi
-  
+
+    if [ ! -n "${SSO_REALM}" ]; then
+      log_warning "Missing SSO_REALM. Defaulting to ${SSO_REALM:-master} realm"
```

Comment:
I think you actually need `SSO_REALM=master` here.  `${SSO_REALM:-master}` doesn't do anything to the value of `SSO_REALM`, it just evaluates to `master` if it is unset or empty.  You should be able to test this by passing an empty value in for `SSO_REALM`, which should yield a broken configuration.

Reply:
@rcernich you're right, I meant to use `:=` instead which sets the variable if empty